anyway/flask_app.py

```diff
@@ -1579,6 +1613,67 @@ def get_role_object(role_name):
     return role
 
 
+@app.route("/admin/update_user", methods=["POST"])
+def admin_update_user() -> Response:
+    allowed_fields = [
+        "user_current_email",
+        "first_name",
+        "last_name",
+        "email",
+        "phone",
+        "user_type",
+        "user_url",
+        "user_desc",
+        "is_user_completed_registration",
+    ]
+
+    res = check_admin_api_with_input(request, allowed_fields)
+    if res:
+        return res
+    reg_dict = request.json
+
+    for field in allowed_fields:
+        if field not in reg_dict:
+            return return_json_error(Es.BR_FIELD_MISSING, field)
+
+    user_current_email = reg_dict.get("user_current_email")
+    if not user_current_email:
+        return return_json_error(Es.BR_NO_EMAIL)
+    if not is_a_valid_email(user_current_email):
+        return return_json_error(Es.BR_BAD_EMAIL)
+    user = get_user_by_email(db, user_current_email)
+    if user is None:
+        return return_json_error(Es.BR_USER_NOT_FOUND, user_current_email)
```

Comment:
Email is not defined as key in `Users` table

Reply:
Yes, OAuth provider don't promise that we will get an email, to solve that we are registering the user in 2 stages - in first stage the user might not have an email but he will have a record in the DB, only in the second stage we will get the user email.

Reply:
How do we know we do not have different users with same email?

Reply:
We only support Google OAuth, Google have a unique id that we are using to differentiate users